doc/user_guide/Parameters.ipynb

```diff
@@ -929,6 +930,80 @@
     "Object().debug_id, SubObject().debug_id"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4a9751f-e307-4b9e-8c25-fc2059200152",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import param"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32cc7b01-55a2-43dd-b886-acf0c30cb5ba",
+   "metadata": {},
+   "source": [
+    "## Arbitrary Metadata\n",
+    "\n",
+    "Parameters may include arbitrary metadata as a mapping stored in the `metadata` attribute (default is `None`). This attribute is not used internally by Param, instead it is meant to enable rich functionality in third-party libraries, or to be leveraged in your own code base to extend Parameters with information that cannot be declared with the other attributes available. Note that a shallow copy of the dictionary you'll pass will be made on class creation and on instance-level parameter creation. This behavior is not specific to the `metadata` attribute, it is how Param handles attributes with value that are mutable containers (lists, dicts, sets)."
```

Comment:
Here it describes the value as a dictionary, whereas above it's described as a mapping. Should presumably be consistent, e.g. by mentioning a dictionary as one possible value rather than the only one expected.